packages/node/test/fixtures/node-no-configuration-composer-with-prefix/services/frontend/index.mjs

```diff
@@ -1,19 +1,20 @@
 import { createServer } from 'node:http'
 
 const server = createServer((req, res) => {
-  if (req.url === '/frontend/') {
+  console.log('Incoming', req.url)
```

Comment:
Leftover?

Reply:
yes! I'll clean it up once I get a green CI.